guides/source/configuring.md

```diff
@@ -2694,24 +2698,25 @@ The default value depends on the `config.load_defaults` target version:
 | (original)            | `false`              |
 | 7.0                   | `true`               |
 
-#### `ActiveSupport::Logger.silencer`
-
-Is set to `false` to disable the ability to silence logging in a block. The default is `true`.
-
-#### `ActiveSupport::Cache::Store.logger`
-
-Specifies the logger to use within cache store operations.
-
-#### `ActiveSupport.to_time_preserves_timezone`
+#### `config.active_support.to_time_preserves_timezone`
 
-Specifies whether `to_time` methods preserve the UTC offset of their receivers. If `false`, `to_time` methods will convert to the local system UTC offset instead.
+Specifies whether `to_time` methods preserve the UTC offset of their receivers or preserves the timezone. If set to `:zone`, `to_time` methods will use the timezone of their receivers. If set to `:offset`, `to_time` methods will use the UTC offset. If `false`, `to_time` methods will convert to the local system UTC offset instead.
 
 The default value depends on the `config.load_defaults` target version:
 
 | Starting with version | The default value is |
 | --------------------- | -------------------- |
 | (original)            | `false`              |
-| 5.0                   | `true`               |
+| 5.0                   | `:offset`            |
```

Comment:
We didn't really change the default value for Rails 5.0, right? This part is confusing.

Reply:
Sorry for late reply (I didn't see it until now).

The value here preserves the backwards compatibility and it behaves how it was with `true` value if it's on 7.1. In a sense, `true` == `:offset`. If you were using 7.1, the default would be `true`, if you were using 7.2 or later, it would be `:offset`.

Reply:
Yeah, you're absolutely right. I like that better.
Not sure where we go from here 😅. Is this worth making a PR?

Reply:
I think the current text is actually correct. The table is not documenting what value a config used to be on previous Rails versions, but what value you get when you `load_defaults X`. If you want to see the values for a previous Rails version you should use a previous version's guide.

Reply:
I hope no one running Rails 8 is using `load_defaults 5.0`